qiskit_nature/algorithms/__init__.py

```diff
@@ -11,31 +11,18 @@
 # that they have been altered from the originals.
 
 """
-Chemistry specific Aqua algorithms (:mod:`qiskit_nature.algorithms`)
+Nature Sciences specific algorithms (:mod:`qiskit_nature.algorithms`)
```

Comment:
Shouldn't it rather by "Natural sciences" or just "Nature" if you want to reflect the repo's name?

Reply:
I think there is going to be quite a lot of documentation cleanup that needs to be done. Manoel really just got rid of Aqua from the text and changed things to build. But it needs going over - since this is all under Nature Sciences now, and are different from Aqua algorithms (before all we had were QuamtumAlgorithm based algos like Aqua, but we did not put them there as they needed params (or were built out internally) from logic that was here.)
Since all these domain repos now have Algorithms etc we can try and make this text consistent across them when we do this work later on the docs.